pkg/controllers/jivavolume_controller.go

```diff
@@ -135,6 +136,16 @@ func (r *JivaVolumeReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return reconcile.Result{}, err
 	}
 
+	if _, ok := podIPMap[instance.Name]; !ok {
+		err = r.updatePodIPMap(instance)
+		if err != nil {
+			// log err only, as controller must be in container creating state
+			// don't return err as it will dump stack trace unneccesary
+			logrus.Infof("failed to get controller pod ip for volume %s: %s", instance.Name, err.Error())
+			time.Sleep(1 * time.Second)
```

Comment:
sleep required ? i get the idea but do we need to have it ?

Reply:
To avoid log flooding with error msgs